### PR TITLE
Update load test workflows

### PR DESCRIPTION
**Pull request by kaneeldias**
## Purpose
$subject 

Related issue: https://github.com/ballerina-platform/ballerina-standard-library/issues/2618

## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: .github/workflows/trigger-load-tests.yml [hunk 1]
```diff
@@ -49,4 +59,4 @@ jobs:
           repo: ballerina-platform/ballerina-performance-cloud
           ref: refs/heads/main
           token: ${{ secrets.BALLERINA_BOT_TOKEN }}
-          inputs: '{ "repo-name": "module-ballerinax-mysql", "tests": "${{ steps.formatTestNames.outputs.testNames }}", "zipURL": ${{ steps.setRuntimeUrl.outputs.runtimeUrl}}, "clusterName": "${{ github.event.inputs.clusterName }}", "dispatchType": "mysql-load-test" }'
+          inputs: '{ "repo-name": "module-ballerinax-mysql", "tests": "${{ steps.formatTestNames.outputs.testNames }}", "zipURL": ${{ steps.setRuntimeUrl.outputs.runtimeUrl}}, "clusterName": "${{ steps.processClusterName.outputs.clusterName }}", "dispatchType": "grpc-load-test" }'
```
Comment (dilanSachi) [inline .github/workflows/trigger-load-tests.yml:62]:
```suggestion
          inputs: '{ "repo-name": "module-ballerinax-mysql", "tests": "${{ steps.formatTestNames.outputs.testNames }}", "zipURL": ${{ steps.setRuntimeUrl.outputs.runtimeUrl}}, "clusterName": "${{ steps.processClusterName.outputs.clusterName }}", "dispatchType": "mysql-load-test" }'
```

File: .github/workflows/trigger-load-tests.yml [hunk 2]
```diff
@@ -11,13 +11,23 @@ on:
...
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+      - name: Process cluster name
+        id: processClusterName
+        run: |
+          clusterName="${{github.event.inputs.clusterName}}"
+          if [ "$clusterName" = "" ]; then
+            clusterName="grpc-perf-cluster-test"
```
Comment (dilanSachi) [inline .github/workflows/trigger-load-tests.yml:28]:
```suggestion
            clusterName="mysql-perf-cluster-test"
```

Comment (kaneeldias) [reply .github/workflows/trigger-load-tests.yml]: Fixed in 54301f0

File: .github/workflows/trigger-load-tests.yml (hunk 1 above)
Comment (kaneeldias) [reply .github/workflows/trigger-load-tests.yml]: Fixed in 54301f0

**Review by dilanSachi — APPROVED** [review]

**Pull request merged**